src/Illuminate/Auth/composer.json

```diff
@@ -14,7 +14,7 @@
         }
     ],
     "require": {
-        "php": "^7.4|^8.0",
+        "php": "^8.0",
```

Comment:
probs should be `^8.0.2`

Reply:
Makes sense since I see now that's also the min PHP version for Symfony 6. I've updated this constraint everywhere.